eclipse-collections-code-generator/src/main/resources/impl/stack/abstractPrimitiveStack.stg

```diff
@@ -256,11 +256,9 @@ public abstract class Abstract<name>Stack implements <name>Stack
     public int hashCode()
     {
         int hashCode = 1;
-        <name>Iterable iterable = this.getDelegate().asReversed();
```

Comment:
I think we should leave the current implementation in AbstractPrimitiveStack and add the new implementation override in the concrete ArrayStack implementations.

Reply:
Happy to make the change, just want to understand the reasoning behind it.  In `AbstactPrimitveStack` `getDelegate()` is defined as returning a `PrimitveArrayList` so the cost of random access of the underlying implementation should not be an issue. Or is it something else?

Reply:
If someone extends the Abstract class, the implementation may not be RandomAccess.